lib/models/ffoi.js

```diff
@@ -0,0 +1,50 @@
+class Ffoi {
+  constructor (s3) {
+    this.s3 = s3
+  }
+  save (file, bucket, key) {
+    let count = file.EATimeSeriesDataExchangeFormat.Station.length
+
+    return new Promise((resolve, reject) => {
+      file.EATimeSeriesDataExchangeFormat.Station.forEach((item) => {
```

Comment:
Could `Promise.all` help here?

It would make the code simpler and mean you don't have to keep track of `count`.


If you go with it as-is, it never calls `reject`. You may want to look at this.